tello Test/tello_web_server.py

```
@@ -651,15 +740,672 @@ def _log_operation(self, operation: str, details: Dict[str, Any]):
+                except:
+                    pass
+                finally:
+                    self.udp_socket = None
+            
+            self.use_ffmpeg = False
+            self.use_simple_udp = False
+            self.latest_frame = None
+            
+            # ビデオストリーミングを無効化
+            if self.is_connected:
+                response = await self._send_command('streamoff')
```

Ruff (0.11.9)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)